.gitignore

```diff
@@ -47,3 +47,4 @@ vendor/**
 /*-success
 .crossdeps
 build/
+.write-test
```

Comment:
I added this after noticing it was left behind when running `travis-worker --queue-type file`.